git-extra/git-sdk.sh

```diff
@@ -20,3 +20,54 @@ case "$rootdir" in
 	fi
 	;;
 esac
+
+sdk () {
+	case "$1" in
+	help|--help|-h)
+		cat >&2 <<-EOF
+		The 'sdk' shell function helps you to get up and running
+		with the Git for Windows SDK. The available subcommands are:
+
+		create-desktop-icon: install a desktop icon that starts the GfW SDK shell.
+
+		init <repo>: initialize and/or update a development repo. Known repos
+		    are: build-extra, git, MINGW-packages, MSYS2-packages.
+
+		build-git:   initializes the Git repo and builds Git.
+		EOF
+		;;
+	create-desktop-icon)
+		create-shortcut.exe --icon-file /msys2.ico --work-dir / /git-bash.exe \
+		"$HOME/Desktop/Git SDK$(case "$(uname -m)" in i686) echo " 32-bit";; x86_64) echo " 64-bit";; esac).lnk"
+		;;
+	die)
+		shift
+		echo "$*" >&2
+		return 1
+		;;
+	init)
+		case "$2" in
+		build-extra|git|MINGW-packages|MSYS2-packages)
+			dest="/usr/src/$2"
+			if test ! -d "$dest"/.git
+			then
+				mkdir -p "$dest" &&
+				git init "$dest" &&
+				git -C "$dest" remote add origin https://github.com/git-for-windows/$2
+			fi &&
```

Comment:
I think I'll want to auto-initialize those worktrees, *without* actually fetching them (as a `git pull` will then Just Work).